pkg/orchestration/run.go

```diff
@@ -58,6 +58,7 @@ func RunAudiusNodes(nodes map[string]conf.NodeConfig, network conf.NetworkConfig
 			audiusdTagOverride,
 		); err != nil {
 			logger.Warnf("Error encountered starting node %s: %s", host, err.Error())
+			logger.Warnf("View full debug log at %s", logger.GetLogFilepath())
```

Comment:
could we merge these two log lines together?

Reply:
Since `err.Error()` could have indefinite length, I figured having the logfile on its own separate line would make it easier see at the end of the output. wdyt?